pkg/tsdb/azuremonitor/azure-log-analytics-table-frame_test.go

```diff
@@ -156,7 +156,7 @@ func loadLogAnalyticsTestFileWithNumber(name string) (AzureLogAnalyticsResponse,
 	var data AzureLogAnalyticsResponse
 
 	path := filepath.Join("testdata", name)
-	f, err := os.Open(path)
+	f, err := os.Open(filepath.Clean(path))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.